content/nginx-one/changelog.md

```diff
@@ -34,6 +34,19 @@ Stay up-to-date with what's new and improved in the F5 NGINX One Console.
 
 ## January 20, 2025
 
+### Manage certificates with Config Sync Groups
+
+With the NGINX One Console, you can now:
+
+Manage certificate deployment in Config Sync Groups. You can:
```

Comment:
```suggestion
With the NGINX One Console, you can now manage certificate deployment in Config Sync Groups. 

You can:
```

The rhythm of the colon and line break read really weird to me. 

Based on the next section "Revert a configuration", there's precedent, but it feels like there should be a list of items following it, not a single one.